content/_index.md

```diff
@@ -10,7 +10,7 @@ specified and known implementations of them.
 
 ## HTTP handler
 
-The [HTTP Handler](http-handler) allows users to write portable HTTP server
```

Comment:
Not sure but I guess this is the idiomatic way to make links with hugo?

https://gohugo.io/content-management/cross-references/

Reply:
yeah I remember this!